src/js/angular/directives/paginations.js

```diff
@@ -0,0 +1,7 @@
+const exportDirectives = angular.module('graphdb.framework.directives.paginations', []);
```

Comment:
This directive was previously in angular/export/directives.js but was bound to the export and namespaces modules at the same time, so it was cleaned up and moved here as it's now a generic one.